saleor/payment/utils.py

```diff
@@ -427,3 +430,49 @@ def price_to_minor_unit(value: Decimal, currency: str):
     number_places = Decimal("10.0") ** precision
     value_without_comma = value * number_places
     return str(value_without_comma.quantize(Decimal("1")))
+
+
+def get_unfinished_payments():
```

Comment:
This function is purely related to querying data from DB - maybe we could rewrite it as a queryset method? It's not a big difference in how the code works, but a bit better organization of the code I think.

Reply:
I thought that we want to keep such things outside a QuerySet class.  This function will be called in one place.

Reply:
The first version was included in queryseet. Now i get out it into separete function. So how it should looks like?

Reply:
let's leave it for now as it is. In the worst scenario, we will move it to the Payment QS in final review.